src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -218,11 +224,11 @@ const PermissionIdentities: FC = () => {
     if (selectedIdentityIds.length > 0) {
       return (
         <SelectedTableNotification
-          totalCount={nonTlsUsers.length ?? 0}
-          itemName="OIDC identity"
+          totalCount={identities.length ?? 0}
```

Comment:
We can only select fine-grained identities for bulk action, the count here should reflect that

Reply:
I have actually made the logic such that we can select in bulk irrespective of the identity type and still delete according to the relevant API. The logic for modifying groups for mixed certificate/oidc bulk selections also works too. I think this would be an easier approach for the User. Do you wish to only select one identity type in bulk at a time?

Reply:
I'm not sure if I follow, fine-grained means oidc entities + new TLS identities. We can't select legacy TLS identities and delete/modify them. The count here represents the total "selectable" identities, so it's not related to the API code which is fine